bin/git-stats-importer

```diff
@@ -51,7 +51,7 @@ myRepo.exec("config user.email", function (err, GIT_EMAIL) {
               , "max-count": "100"
               , skip: c.toString()
               , all: true
-            }, "log");
+            }, "log --no-color");
```

Comment:
Instead of adding it here, please pass it into the object. Also, check again if it's working properly.  [Check out the docs here](https://github.com/IonicaBizau/node-oargv#oargvoptions-prgm).

```js
var command = OArgv({
    author: GIT_EMAIL
  , "max-count": "100"
  , skip: c.toString()
  , all: true
  , "no-color": true
}, "log");
```